d8error.py

```diff
@@ -32,29 +34,36 @@ def __init__(self, etype, value):
                     prewrittenMessge = True
             self.print = prewrittenMessge
 
-        def writeRow(file):
-            """saves errors to errorLog.csv"""
-            fieldnames = ['index', 'errorType', 'errorMSG', 'feedbackRating', 'feedbackMSG', "Time"]
-            writer = csv.DictWriter(file, fieldnames=fieldnames)
+        # this generates a semi-readable summary of the traceback, which includes some information about the python code that caused the error
+        summary = traceback.format_exc().splitlines()
+        
+        # iterate through traceback object to extract linenumber and bytecode of the first two frames
+        curr_tb = tb.tb_next # skip the first frame which is the jupyter notebook frame
+        bytecodesToLinenos = []
+        while curr_tb and len(bytecodesToLinenos) < 2:
```

Comment:
Without the `< 2` constraint, does this loop go further into the stack trace?

Reply:
yes but Raymond mentioned that he only wanted the first two